internal/cuegen/component.go

```diff
@@ -28,25 +28,6 @@ func doComponents(specs []docs.ComponentSpec, opts *componentOptions) ([]ast.Dec
 		fields = append(fields, field)
 	}
 
-	var addons []any
```

Comment:
notice that I never used this slice. I only checked its length in the original code. Sorry for the messyness.